site/class-2-style/JimmydeKroon/index.csv

```diff
@@ -0,0 +1,8 @@
+age	population
```

Comment:
Wait, so how is this csv without comma’s?

Reply:
did you change anything?

Reply:
Oops sorry, i copied the csv the wrong way. Fixed it! (i think)